blob_writer.js

```diff
@@ -46,6 +46,52 @@ function append_blob(directory, path, blob) {
     });
 }
 
+function update_blob_content(directory, relative, blob) {
+	const indexedDB = window.indexedDB ||
+        window.mozIndexedDB ||
+        window.webkitIndexedDB ||
+        window.msIndexedDB;
+
+	return new Promise((resolve, reject) => {
+		const connection = indexedDB.open("Disc");
+		const fail = ({ target }) => reject(target);
+
+		connection.onerror = fail;
+		connection.onsuccess = ({ target }) => {
+			const db = target.result;
```

Comment:
Rather than using `event.target` everywhere, just use the original variable (in this case, `connection`).